kitsune/users/management/commands/user_maintenance.py

```diff
@@ -0,0 +1,44 @@
+from django.core.management.base import BaseCommand
+
+from datetime import datetime, timedelta
+
+from django.contrib.auth.models import User
+from kitsune.users.utils import anonymize_user
+
+
+class Command(BaseCommand):
```

Comment:
Instead of using a command we can use a celery task. This will make the transition easier when we remove the APScheduler.